aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -557,6 +561,30 @@ namespace {
     }
   }
 
+  cudnnRNNAlgo_t get_algo(const RNNDescriptorParams& rnn, const TensorDescriptorListParams& tensors){
+#if CUDNN_VERSION < 7200 || CUDA_VERSION < 9010
+      return CUDNN_RNN_ALGO_STANDARD;
+#else
+      cudaDeviceProp* prop = at::cuda::getCurrentDeviceProperties();
```

Comment:
So, we should treat this condition as empirically determined on V100s?

Reply:
Yes. Persistent rnns provide benefits on other cards too, but for much smaller batches/longer sequences.